docs/client/security.rst

```diff
@@ -188,22 +181,13 @@ Authentication errors
 ^^^^^^^^^^^^^^^^^^^^^
 
 The operation methods of :class:`~pywbem.WBEMConnection` raise
-:exc:`pywbem.AuthError` in any of these situations:
-
-* When client side verification of the X.509 server certificate fails.
-
-* When the WBEM server returns HTTP status 401 "Unauthorized" and the
-  retries in the client are exhausted. The server typically returns
-  that status in any of these situations:
-
-  - no authorization information provided by client
-  - wrong HTTP authentication scheme used by client
-  - authentication failed
-  - user is not authorized to access resource
-
-.. _`Default CA certificate paths`:
-
-Default CA certificate paths
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. autodata:: pywbem.cim_http.DEFAULT_CA_CERT_PATHS
+:exc:`pywbem.AuthError` when the WBEM server returns HTTP status 401
+"Unauthorized" and the retries in the client are exhausted. The server
```

Comment:
Should we mention retries in the client.  We removed that code and delegated any retries to requests and its subpackages.

Reply:
It is true that the retries are done in the "requests" package or its sub-packages,  but that is still in the WBEM client. I think the user ought to understand that the WBEM client performs retries.